website/docs/r/app_service_certificate_order_key_vault_store.html.markdown

```diff
@@ -0,0 +1,131 @@
+---
+subcategory: "App Service (Web Apps)"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_certificate_order_key_vault_store"
+description: |-
+  Manages an App Service Certificate Order Certificate.
+
+---
+
+# azurerm_app_service_certificate_order_key_vault_store
+
+Manages an App Service Certificate Order Certificate.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+data "azurerm_client_config" "test" {}
+
+resource "azurerm_key_vault" "test" {
+  name                = "example-keyvault"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  tenant_id = data.azurerm_client_config.test.tenant_id
+
+  sku_name = "standard"
+
+  access_policy {
+    tenant_id = data.azurerm_client_config.test.tenant_id
+    object_id = "f8daea97-62e7-4026-becf-13c2ea98e8b4"
+
+    secret_permissions = [
+      "Delete",
+      "Get",
+      "Purge",
+      "Set",
+      "List"
+    ]
+
+    certificate_permissions = [
+      "Create",
+      "Delete",
+      "Get",
+      "Purge",
+      "Import",
+      "List"
+    ]
+  }
+
+  access_policy {
+    tenant_id = data.azurerm_client_config.test.tenant_id
+    object_id = "ed47c2a1-bd23-4341-b39c-f4fd69138dd3"
```

Comment:
Same here, can you please replace this with an appropriate reference

Reply:
I will avoid using the real value next time.